### PR TITLE
Feat/actual content

### PR DESCRIPTION
**Pull request by ikbensiep**
WIP

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,28 +4,363 @@
...
+      <section id="about">
+        <h2><em>Minimal</em> code for <strong>maximum</strong> result.</h2>
+        <h3>Meet the <strong>team</strong></h3>
+        <div class="flex-row team">
+          <div class="person card">
+            <figure>
+              <img src="img/team/evert-color.png" />
+            </figure>
+            <div class="socials">
+              <a href="//twitter.com/evertp" target="_blank" rel="noopener noreferrer">
```
Comment (evert) [inline index.html:177]: These should all be `https://` not protocol-relative urls! Would also drop `noopener noreferrer`, no harm to keep that

File: index.html [hunk 2]
```diff
@@ -4,28 +4,363 @@
...
+      <!-- about -->
+      <section id="about">
+        <h2><em>Minimal</em> code for <strong>maximum</strong> result.</h2>
+        <h3>Meet the <strong>team</strong></h3>
+        <div class="flex-row team">
+          <div class="person card">
+            <figure>
+              <img src="img/team/evert-color.png" />
+            </figure>
+            <div class="socials">
```
Comment (evert) [inline index.html:176]: Could we add my (and other's) personal websites? Probably the most important link for me!

File: index.html [hunk 3]
```diff
@@ -4,28 +4,363 @@
...
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/toronto-logo.svg" alt="City of Toronto logo"></a>
+            <small>City of Toronto</small>
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/underknown-logo-white.svg" alt="Underknown logo"></a>
+            <small>Underknown</small>
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/zerolens-white.svg" alt="Zerolens logo"></a>
```
Comment (evert) [inline index.html:274]: Zerolens has been renamed to 'Colorful', so we should probably use that: https://www.colorful.app/

File: index.html [hunk 4]
```diff
@@ -4,28 +4,363 @@
...
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/wealthsimple.svg" alt="Wealthsimple logo"></a>
+            <small>Wealthsimple</small>
+          </li>
+        </ul>
+      </section>
+      
+      <!-- contact -->
+      <section id="contact">
```
Comment (evert) [inline index.html:285]: Could we add 2 things to this section:

1. The email address, for people that prefer sending an email over sending a form.
2. Calendly link: https://calendly.com/evert-bg/30min

**Review by evert — COMMENTED** [review]
Looking good! this is getting really close to being ready!

**Review by BeckyPollard — COMMENTED** [review]
I AM SO EXCITED FOR THIS PROJECT! You've done a fantastic job!!

Also, absolutely loving the b/w faded portraits. It's like the founders are powerful, dithered spirits helping us from beyond. Sometimes I think I can still hear their Element message pings~~~

**Review by evert — APPROVED** [review]

**Pull request merged**